js/src/main/scala/org/soundsofscala/instrument/Instrument.scala

```diff
@@ -23,12 +23,23 @@ import org.soundsofscala
 import org.soundsofscala.models
 import org.soundsofscala.models.*
 
-trait Instrument:
+trait Instrument[Settings](using Default[Settings]):
```

Comment:
I'm a bit suspicious of the `using` clause, as the given value is not used anywhere within the body. I recall we added it to ensure that `Settings` has a `Default` instance, but I feel that probably isn't necessary.

I don't have a suggestion for a change here, just a feeling that it probably could be removed.